metadeploy/api/models.py

```diff
@@ -956,7 +956,15 @@ class ScratchOrg(HashIdMixin, models.Model):
 
     objects = ScratchOrgQuerySet.as_manager()
 
+    def clean_config(self):
+        if self.config:
+            banned_keys = {"email"}
```

Comment:
I'd just move `banned_keys` out of the conditional, as a regular class attr. Otherwise LGTM